emsurfer/README.md

```diff
@@ -0,0 +1,27 @@
+# Emsurfer
```

Comment:
[red]
Maybe have a small description instead of the auto generated one. Something like
# EM-Surfer
This directory contains the ng app used to manager EM-Surfer search requests.

Reply:
Done